packages/hmr/rollup.config.ts

```diff
@@ -1,30 +1,29 @@
 import sucrase from '@rollup/plugin-sucrase';
+import type { Plugin, RollupOptions } from 'rollup';
 
 const plugins = [
+  // @ts-expect-error I don't know why error happening here
   sucrase({
     exclude: ['node_modules/**'],
     transforms: ['typescript'],
   }),
-];
+] as Plugin[];
 
-/**
- * @type {import("rollup").RollupOptions[]}
- */
 export default [
   {
     plugins,
     input: 'lib/injections/reload.ts',
     output: {
-      format: 'iife',
-      file: 'build/injections/reload.js',
+      format: 'esm',
+      file: 'dist/lib/injections/reload.js',
     },
   },
   {
     plugins,
     input: 'lib/injections/refresh.ts',
     output: {
-      format: 'iife',
-      file: 'build/injections/refresh.js',
+      format: 'esm',
+      file: 'dist/lib/injections/refresh.js',
     },
   },
-];
+] as RollupOptions[];
```

Comment:
would be better to use `satisfy` over `as` here
```suggestion
] satisfy RollupOptions[];
```
then  the type will be still validated instead on overwriting the type

Reply:
Thanks